utils/wasm-builder/src/lib.rs

```diff
@@ -82,6 +82,20 @@ impl WasmBuilder {
         self
     }
 
+    /// Add check of recommended toolchain.
+    pub fn with_recommended_toolchain(mut self) -> Self {
+        self.cargo.set_check_recommended_toolchain(true);
+        self
+    }
+
+    /// Force the recommended toolchain to be used, but do not check whether the
+    /// current toolchain is recommended.
+    #[doc(hidden)]
+    pub unsafe fn with_forced_recommended_toolchain(mut self) -> Self {
```

Comment:
I don't see actual usage. Can be removed because check throws error and thus forces to use recommended nightly

Reply:
@StackOverflowExcept1on isn't it a time to already use the method for `out-of-memory` demo, for example?